package.json

```diff
@@ -25,12 +25,11 @@
   "license": "MIT",
   "scripts": {
     "start": "react-native start",
-    "start:android": "cd example && yarn react-native run-android",
-    "start:ios": "cd example && yarn react-native run-ios --project-path `pwd`/ios",
-    "start:macos:packager": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos",
+    "start:android": "react-native run-android",
```

Comment:
```

info Starting the app on "emulator-5554"...
Starting: Intent { cmp=com.microsoft.reacttestapp/.MainActivity }
Error type 3
Error: Activity class {com.microsoft.reacttestapp/com.microsoft.reacttestapp.MainActivity} does not exist.
```

--> **Error: Activity class {com.microsoft.reacttestapp/com.microsoft.reacttestapp.MainActivity} does not exist.**

:thinking: